static/js/message_edit.js

```diff
@@ -515,7 +515,6 @@ exports.end = function (row) {
     if (row !== undefined) {
         current_msg_list.hide_edit_topic(row);
     }
-    condense.show_message_expander(row);
```

Comment:
Removing this line is a behavior change and maybe a bug -- if you have a condensed message and hit `e` followed by `escape`, it ends up uncondesned.